src/safebfuns.c

```diff
@@ -14,7 +14,9 @@
 
 void explicit_bzero( void * buf, size_t n ) {
 	volatile char * volbuf = ( volatile char * ) buf;
-	for( size_t i = 0; i < n; i++ ) {
+
+	size_t i;
+	for (i = 0; i < n; i++) {
```

Comment:
Preserve original spacing style:
```suggestion
	for( i = 0; i < n; i++ ) {
```